stdlib/public/core/Duration.swift

```diff
@@ -99,6 +99,36 @@ extension Duration {
   }
 }
 
+@available(SwiftStdlib 6.0, *)
+extension Duration {
+  /// The duration represented in attoseconds.
```

Comment:
I think that this summary string isn't quite right. Probably something like "the number of attoseconds represented by this Duration". @natecook1000 might have a good suggestion.

Reply:
Yeah, appreciate the review for the docs. I am not a native speaker and to be honest writing the docs took me way more time than implementing this...